src/sphinxcontrib/towncrier/_towncrier.py

```diff
@@ -16,6 +16,12 @@
         load_config_from_file,
     )
 
+try:
+    from dataclasses import asdict, is_dataclass  # noqa: WPS433
+except ImportError:
+    # pylint: disable=unnecessary-lambda-assignment
+    is_dataclass = lambda _: False  # noqa:  WPS440
```

Comment:
This pylint comment could be avoided by doing:
```suggestion
except ImportError:
    def is_dataclass(obj: Any) -> bool:  # noqa: WPS440, WPS110, W0613
        """Return always `False`, as it will never be a dataclass."""
        return False
```
The error codes ignored are :
- WPS440 ("Found block variables overlap: is_dataclass"): Cant really fix this one, we want to do this exact thing
- WPS110 ("Forbid blacklisted variable names."): `obj` seems to be a blacklisted name, but if we change it mypy will complain, so best to just ignore it 
- W0613 ("Unused argument 'obj' (unused-argument)"): Could be fixed by using `_`, but then mypy complains because the signatures dont match. I think its best to have a flake8 ignore then

Reply:
But in your case 3 warnings  should be ignored instead of 2

Reply:
Marking this as resolved, since I'm refactoring this branch to stop relying on this helper.